### PR TITLE
[PAT-1371] add release script & app version

### PR DESCRIPTION
**Pull request by ivanjiang5628**
## Description
- [Jira Ticket](https://jira-jane.atlassian.net/browse/PAT-1371)

- Adds release script
- Adds app client version to the webpack config script & analytics event

### Release Risk Assessment
Low Risk

### Fixed / Expected Behaviour
- Run the command: `sh version_release.sh`. This should generate a release PR and automatically increment the version number.
image

- Ensure that the analytics event parameters include the app_version field.
image

### After

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -177,5 +177,6 @@
     "validate": "npm ls",
     "start": "make dev",
     "android": "react-native run-android"
-  }
+  },
+  "webAppVersion": "23.10.1.0"
```
Comment (ivanjiang5628) [inline package.json:181]: Set default version to "23.10.0.0" to prevent merge conflicts.

File: react/features/app/functions.native.js [hunk 2]
```diff
@@ -56,3 +56,12 @@ export function getBrowserSessionId() {
...
     return DeviceInfo.getVersion();
 }
+
+/**
+ * Returns native app version defined by Jane.
+ *
+ * @returns {string}
+ */
+export function getClientVersion() {
+    return getAppVersion();
```
Comment (ivanjiang5628) [inline react/features/app/functions.native.js:66]: For now, we don't want to inject the web app's version number into the iOS app, as the iOS app already maintains its own versioning in the App Store.

File: react/features/analytics/handlers/JaneHandler.js [hunk 3]
```diff
@@ -42,7 +43,8 @@ export default class JaneHandler extends AbstractHandler {
             'participant_id': participant.participant_id,
             'participant_type': participant.participant_type,
             'uid': this.uid,
-            'browser_session_id': this.browserSessionId
+            'browser_session_id': this.browserSessionId,
+            'app_version': this.jitsiClientVersion
```
Comment (ivanjiang5628) [inline react/features/analytics/handlers/JaneHandler.js:47]: Injecting the version to analytics events.

**Review by bitghost2m — APPROVED** [review]

**Comment by ivanjiang5628** [comment]
## Test Plan

- [x] Ensure that the analytics event parameters include the app_version field.
image
image

## Browsers

- [x] Chrome
- [x] Safari
- [x] Firefox
- [x] Android Chrome

## Pass PR in GitHub / Mark Scheduled for Release in Linear

- [x]  Mark as "Ready for Release"

**Pull request merged**